src/huggingface_hub/inference/_generated/types/text_classification.py

```diff
@@ -0,0 +1,43 @@
+# Inference code generated from the JSON schema spec in @huggingface/tasks.
+#
+# See:
+#   - script: https://github.com/huggingface/huggingface.js/blob/main/packages/tasks/scripts/inference-codegen.ts
+#   - specs:  https://github.com/huggingface/huggingface.js/tree/main/packages/tasks/src/tasks.
+from dataclasses import dataclass
+from typing import Literal, Optional
+
+from .base import BaseInferenceType
+
+
+ClassificationOutputTransform = Literal["sigmoid", "softmax", "none"]
+
+
+@dataclass
+class TextClassificationParameters(BaseInferenceType):
+    """Additional inference parameters
+    Additional inference parameters for Text Classification
+    """
+
+    function_to_apply: Optional["ClassificationOutputTransform"]
+    top_k: Optional[int]
+    """When specified, limits the output to the top K most probable classes."""
+
+
+@dataclass
+class TextClassificationInput(BaseInferenceType):
+    """Inputs for Text Classification inference"""
+
+    inputs: str
+    """The text to classify"""
+    parameters: Optional[TextClassificationParameters]
+    """Additional inference parameters"""
+
+
+@dataclass
+class TextClassificationOutput(BaseInferenceType):
+    """Outputs of inference for the Text Classification task"""
+
+    label: str
+    """The predicted class label."""
+    score: float
+    """The corresponding probability."""
```

Comment:
Output is actually a List of those if I'm not mistaken :(

I had to manually parse the TS code to add that array definition in huggingface.js (bug also reported in quicktype)

Reply:
Yeah I already had a script to handle the shared classes (typically `ClassificationOutput`) but now I pushed https://github.com/huggingface/huggingface_hub/pull/2036/commits/56955a87d69fe4be01ffd67a617f68f655b72662 so that classes are named `AudioClassificationOutputElement`, to emphasize that the expected output will be a list of those. 

Is it better for you like this?